analytics.js

```diff
@@ -176,9 +178,10 @@ class Analytics {
       // logger.debug("this.clientIntegrations: ", this.clientIntegrations)
       // Load all the client integrations dynamically
       this.clientIntegrations.forEach((intg) => {
-        const modName = configToIntNames[intg.name];
+        const scriptName = configToIntNames[intg.name];
+        const modName = `${scriptName}${INTG_SUFFIX}`;
```

Comment:
For the sake of avoiding confusion, let's keep the naming this way:

```
const modName = configToIntNames[intg.name]; // script URL can be constructed from this
const pluginName = `${modName}${INTG_SUFFIX}`; // this is the name of the object loaded on the window
```

Reply:
Done